src/clients/translation/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
nit: I think of clients as interacting with services outside of the browser, with that definition this wouldn't belong here. The translation code is spread between here and assets when they could alternatively be grouped in one folder.

Main takeaway though is what is the definition of the clients folder?

Reply:
Got you. About the translations, I hesitated but then I figured since we place all the images used in different pages/components inside the `assets` folder, then translations would technically receive the same treatment since they are the texts contained in all pages/components.
That said, I don't mind having them inside the client too.

I see clients as local libraries personally, so they could contain anything: functions, classes, hooks, components etc... In this case, I'd rather have the `Trans` component inside the client itself, because it relies on the `Trans` component imported from i18-next. This way, the usage of i18next is restricted to our translation client only, making it easier to replace in the future if needed.